### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by sushantdhiman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Sequelize is a promise-based Node.js/io.js ORM for Postgres, MySQL, SQLite and M
 
 From 3.0.0 and up Sequelize will follow SEMVER. 3.0.0 contains important security fixes so we highly recommend that users upgrade.
 
-If you still use 1.7 please prefer to [Upgrading to 2.0](https://github.com/sequelize/sequelize/wiki/Upgrading-to-2.0) and the changelog between 2.0 and 3.0. 2.1 also has a breaking change.
+Starting from 4.0.0 Sequelize will only support Node v4 and above to use ES6 features. If you are still using older versions like 1.7 or 2.0, Please consider updating to latest v3.
```
Comment (mickhansen) [inline README.md]: Can probably kill mentions of 1.7 and 2.0

Comment (sushantdhiman) [inline README.md]: Notice still mentions them as older version 1.7 or 2.0, Do we really need them here, considering we don't provide support for them anymore.

Comment (mickhansen) [inline README.md]: No they can be removed

Comment (sushantdhiman) [inline README.md]: Great

**Pull request merged**